tests/unit/test_charm.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+from unittest.mock import PropertyMock, patch
+
+from charm import SlurmdCharm
+from ops.model import ActiveStatus, BlockedStatus
+from ops.testing import Harness
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
I don't think we need to specify the return type if it is None.

Reply:
I prefer to see return types on functions that return `None`, if nothing else, for clarity and consistency.

Reply:
This method doesn't explicitly return None, though. We haven't typically used a None return type in these cases for our other code bases.

Reply:
100%